pandas/core/common.py

```diff
@@ -386,7 +274,7 @@ def _get_callable_name(obj):
     return None
 
 
-def _apply_if_callable(maybe_callable, obj, **kwargs):
+def apply_if_callable(maybe_callable, obj, **kwargs):
```

Comment:
prob should move all of the indexing ones to a new indexing module

maybe pandas.core.indexing.utils
(and obviously move indexing.py itself)